src/app/modules/organization-management/components/organizations-table/organizations-table.component.spec.ts

```diff
@@ -43,7 +43,7 @@ describe('OrganizationsTableComponent', () => {
   let fixture: ComponentFixture<OrganizationsTableComponent>
   let router: Router
 
-  const organizationsSubject$ = new Subject<IOrganization[]>()
+  const organizationsSubject$ = new Subject<any>()
```

Comment:
Is this inevitable? We should only do this if there is no other way to keep the types.

Reply:
I tried to change this, but I am afraid it can not be changed. When paginated, that typed is changed